ChatQnA/docker_compose/intel/hpu/gaudi/README.md

```diff
@@ -95,504 +104,169 @@ curl http://${host_ip}:8888/v1/chatqna \
+ ✔ Container dataprep-redis-server         Removed                                                                                                 10.5s
+ ✔ Container chatqna-gaudi-ui-server       Removed                                                                                                 10.3s
+ ✔ Container chatqna-gaudi-backend-server  Removed                                                                                                 10.3s
+ ✔ Container vllm-gaudi-server             Removed                                                                                                  0.0s
+ ✔ Container retriever-redis-server        Removed                                                                                                 10.4s
+ ✔ Container tei-reranking-gaudi-server    Removed                                                                                                  2.0s
+ ✔ Container tei-embedding-gaudi-server    Removed                                                                                                  1.2s
+ ✔ Container redis-vector-db               Removed                                                                                                  0.4s
+ ✔ Network gaudi_default                   Removed                                                                                                  0.4s
 ```
 
-Then run the command `docker images`, you will have the following 5 Docker Images:
-
-- `opea/retriever:latest`
-- `opea/dataprep:latest`
-- `opea/chatqna:latest`
-- `opea/chatqna-ui:latest`
-- `opea/nginx:latest`
-
-If Conversation React UI is built, you will find one more image:
+The ChatQnA containers should be stopped and removed after execution finishes.
 
-- `opea/chatqna-conversation-ui:latest`
+## ChatQnA Docker Compose Files
 
-If Guardrails docker image is built, you will find one more image:
+In the context of deploying a ChatQnA pipeline on an Intel Gaudi platform, the allocation and utilization of Gaudi devices across different services are important considerations for optimizing performance and resource efficiency. Each of the four example deployments, defined by the example Docker compose yaml files, demonstrates a unique approach to leveraging Gaudi hardware, reflecting different priorities and operational strategies.
 
-- `opea/guardrails:latest`
+### compose.yaml - Default Deployment
 
-## 🚀 Start MicroServices and MegaService
+The default deployment utilizes Gaudi devices primarily for the `vllm-service`, which handles large language model (LLM) tasks. This service is configured to maximize the use of Gaudi's capabilities, potentially allocating multiple devices to enhance parallel processing and throughput. The `tei-reranking-service` also uses Gaudi hardware (1 card), however, indicating a balanced approach where both LLM processing and reranking tasks benefit from Gaudi's performance enhancements.
```

Comment:
may want them to note the name "gaudi" in the container name ghcr.io/huggingface/tei-gaudi:1.5.0 opea/vllm-gaudi:latest